docs/conf.py

```diff
@@ -94,13 +94,19 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = "sphinx"
 
+# Add any paths that contain custom static files (such as style sheets) here,
+# relative to this directory. They are copied after the builtin static files,
+# so a file named "default.css" will overwrite the builtin "default.css".
+html_static_path = ["_static", "logos"]
+
 
 # -- Options for HTML output -------------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
 html_theme = "pydata_sphinx_theme"
+html_favicon = "favicon.ico"
```

Comment:
```suggestion
html_favicon = "logos/favicon.ico"
```

I don't know why but the favicon is not relative to the static path 🤷🏿